docs/admin/kubelet-tls-bootstrapping.md

```diff
@@ -0,0 +1,96 @@
+---
+assignees:
+- mikedanese
+
+---
+
+* TOC
+{:toc}
+
+## Summary
+
+This document describes setting up TLS client certificate bootstrapping for kubelets.
+Kubernetes 1.4 introduces an experimental API for requesting certificates from a cluster-level 
```

Comment:
I'll leave that as a follow up for @mikedanese, this PR just moved the content to its own file